shardingsphere-sql-parser/shardingsphere-sql-parser-binder/src/main/java/org/apache/shardingsphere/sql/parser/binder/segment/select/projection/impl/DerivedProjection.java

```diff
@@ -38,6 +39,8 @@
     
     private final String alias;
     
+    private final SQLSegment realProjection;
```

Comment:
what mean of realProjection? For my understanding, derived projection do not have `real projection`, can you rename it as correct name?